src/games/strategy/triplea/Constants.java

```diff
@@ -144,6 +144,7 @@
   String UNIT_TYPE_CARRIER = "carrier";
   String UNIT_TYPE_FIGHTER = "fighter";
   String UNIT_TYPE_BOMBER = "bomber";
+  String UNIT_TYPE_TAC_BOMBER = "tactical_bomber";
```

Comment:
This value is only used in TestUtil, please move it there.
Not only that, the 'Constants.java' is not a very good design, I'd like to avoid adding to it.